apt-hook/hook.cc

```diff
@@ -140,7 +151,7 @@ static void check_esm_upgrade(pkgCache::PkgIterator pkg, pkgPolicy *policy, resu
       {
          if (pf.File().Archive() != 0 && pf.File().Origin() == std::string("UbuntuESM"))
          {
-            res.esm_i_packages.push_back(pf.File().FileName());
+            res.esm_i_packages.push_back(pkg.Name());
```

Comment:
Found a bug here where we were actually rendering the apt policyfile names instad of pkg name.

Reply:
Oop! Thought I fixed that but clearly didn't commit it - nice catch!